data/src/main/java/entities/JPAPOIEntity.java

```diff
@@ -0,0 +1,52 @@
+package entities;
+
+import enums.POITypes;
+
+import javax.persistence.*;
+
+
+@Entity
+@Table(name = "Object")
```

Comment:
Why table called Object? in code we have POI - maybe it's better to have the same names in DB and code